### PR TITLE
Adding language support for Germany

### PR DESCRIPTION
**Pull request by Skyper1987**
This is my first translation of the US language file to German. I would like to check it out in the Privacy Badger so I can optimize the translation.

### CODE REVIEW

**Comment by cooperq** [comment]
Excellent work. Thank you very much!

File: locale/de.properties [hunk 1]
```diff
@@ -0,0 +1,69 @@
+activate_on_site = Klicke hier um den Privacy Badger auf dieser Seite zu aktiveren.
+click_badger_activate_on_site = Klicke auf das Dachs-Symbol um den Privacy Badger auf dieser Seite zu aktivieren.
+cookie_prefs_warning = <p id="cookiePrefsWarning">
+Achtung: Der Privacy Badger Alpha arbeitet vielleicht nicht wie erwartet mit
```
Comment (rugk) [inline locale/de.properties]: At first I would not add an article before "Privacy Badger". Privacy Badger is a proper name.
I also don't know why it is still called "alpha" there.

File: locale/de.properties [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+<a href="https://support.mozilla.org/en-US/kb/enable-and-disable-cookies-website-preferences" target="_blank">
+mit deinen Drittanbieter Cookie Einstellungen</a>. Du findest vielleicht einen Fehler den Du uns mitteilen kannst <a href="https://github.com/EFForg/privacybadgerfirefox/issues/" target="_blank">here</a>. Danke für deine Hilfe!</p>
```
Comment (rugk) [inline locale/de.properties]: "here" should be translated and not a this position of the sentence.

File: locale/de.properties [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+remove_button = Webseite entfernen
+options_slider_text = Bewege die Schieber um Webseiten zu sperren/entsperren
+options_loading = Lade...
+options_social_enable = Aktiviert das Blocken von Social Widgets
+firstrun_by_eff = Ein Projekt der Electronic Frontier Foundation
+firstrun_title = Danke das Du den Privacy Badger installiert hast!
+firstrun_social_share = Lass die Welt wissen das Du dich gegen Online Tracking wehrst.
+slideshow_1 = Hallo, ich bin der Privacy Badger! Danke das Du mich installiert hast. Damit wir gut zusammenarbeiten können, möchte ich dir ein paar hilfreiche Tipps geben.
+slideshow_2 = Ich werde hier oben in deinen Browser nach Werbung und Trackern Ausschau halten, die verfolgt was du im Internet machst.
+slideshow_3 = Wenn ich feststelle das etwas versucht dich zu verfolgen verhindere ich das.
```
Comment (rugk) [inline locale/de.properties]: dich zu verfolgen**,** verhindere ich das. (AFAIK)

File: locale/de.properties [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+slideshow_9 = Ich lerne während ich arbeite. Umso erfahrener ich bin desto besser schütze ich deine Privatsphäre.
+slideshow_10 = Wenn Du denkst das ich eine falsche Entscheidung für einen Tracker getroffen habe, kannst Du diese manuell korrigieren. Bewege einfach die Schieber: rot um zu blockieren, gelb um teilweise zu blockieren und grün um zu erlauben.
+slideshow_11 = Wenn Du eine meiner Entscheidungen verändert hast, kannst Du sie jederzeit wieder zurücksetzen. Klicke einfach auf den Zurücksetzen Pfeil.
+slideshow_12 = Wenn Du Probleme mit einer Webseite hasst und denkst das ich daran Schuld bin, kannst Du hier auf den Deaktivieren Knopf drücken. Ich werde dann schlafen und auf der Webseite nichts mehr blockieren.
+slideshow_13 = Sollte ich Probleme mit einer Webseite verursacht haben informiere bitte meine Entwicker. Sie können dann an einer Lösung arbeiten. Klicke einfach auf diesen Knopf.
+slideshow_14 = Wenn du weitere Einstellungen ändern möchtest klicke auf das Zahnrad Symbol.
```
Comment (rugk) [inline locale/de.properties]: "ändern möchtest**,** klicke" (AFAIK)

File: locale/de.properties [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+slideshow_15 = Natürlich kannst Du die Einstellungen auch unverändert lassen. Ich werde dann still im Hintergrund alle Cookies fressen die ich erkenne.
+slideshow_16 = Danke das Du diesen wichtigen Schritt zum Schutz deiner Privatsphäre getan hast. Solltest Du einmal Hilfe brauchen kannst du hier klicken. Klicke weiter unten um der Welt zu sagen das Du dich gegen Online Tracking wehrst.
```
Comment (rugk) [inline locale/de.properties]: "Danke**,*** das**s** Du"

Comment (rugk) [inline locale/de.properties]: "Hilfe brauchen**,** kannst"

Comment (rugk) [inline locale/de.properties]: "weiter unten**,** um" (AFAIK)

Comment (rugk) [inline locale/de.properties]: "sagen**,** das**s** Du"

File: locale/de.properties [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+slideshow_3 = Wenn ich feststelle das etwas versucht dich zu verfolgen verhindere ich das.  
+slideshow_4 =  Wenn du auf mein Symbol klickst kannst Du dir eine Liste der im aktuellen Tab erkannten potentiellen Tracker anzeigen lassen.
+slideshow_5 = Ist der Schieber rot, dann habe ich den Tracker komplett gesperrt.
+slideshow_6 = Ist der Schieber gelb, dann habe ich den Tracker teilweise gesperrt. Ich werde mein Bestes tun um dich vor Verfolgung zu schützen. Allerdings kann das Blockieren des Trackers zu Problemen bei einigen Webseiten führen.
+slideshow_7 = Ist der Schieber grün konnte ich keine Verfolgung durch die Webseite feststellen. Ich werde trotzdem ein Auge drauf haben. 
+slideshow_8 = Wenn Du das erste Mal auf mein Symbol klickst siehst Du vielleicht das alle Tracker grün sind – keine Sorge! Ich unterscheide mich von anderer Trackingschutz Software. Um Tracker zu blockieren verlasse ich mich nicht auf Listen. Stattdessen lerne ich während Du surfst. Falls ich aktuell noch nichts blockiere öffne noch 4 oder 5 verschiedene Webseiten. Danach sollte ich gewöhnliche Tracker blockieren.
```
Comment (rugk) [inline locale/de.properties]: "Symbol**,** klickst"

Comment (rugk) [inline locale/de.properties]: "vielleicht**,** das**s**"

**Comment by rugk** [comment]
@Skyper1987 http://www.das-dass.de/ :wink: 

If you don't want to change the things I noted by yourself, I could also create a Pull Request.

**Comment by Skyper1987** [comment]
Hello rugk,
Thank you for your audit. I've seen today that the german translation is finally integrated through the Privacy Badger. No I will optimize the translation and allow your changes. But I don't agree to your meaning about the article before the name "Privacy Badger". In my opinion we talk about "the" badger (der Privatsphären Dachs). "Klicke hier um Dachs zu deaktivieren". Sounds a bit wrong in my opinion.
Regards,
Skyper1987

File: locale/de.properties (hunk 3 above)
Comment (rugk) [inline locale/de.properties]: @Skyper1987 said:
> But I don't agree to your meaning about the article before the name "Privacy Badger". In my opinion we talk about "the" badger (der Privatsphären Dachs). "Klicke hier um Dachs zu deaktivieren". Sounds a bit wrong in my opinion.

Comment (rugk) [inline locale/de.properties]: Yes it sounds a bit weird in both ways, but as the text seems to be a "personal" introduction of the badger I think it is okay to say "the".

File: locale/de.properties (hunk 1 above)
Comment (rugk) [inline locale/de.properties]: @Skyper1987 said:
> But I don't agree to your meaning about the article before the name "Privacy Badger". In my opinion we talk about "the" badger (der Privatsphären Dachs). "Klicke hier um Dachs zu deaktivieren". Sounds a bit wrong in my opinion.

Comment (rugk) [inline locale/de.properties]: Yes it sounds a bit weird in both ways, but as the text seems to be a "personal" introduction of the badger I think it is okay to say "the".

**Comment by rugk** [comment]
BTW you can comment inline when you want to discuss a specific comment of me.

**Pull request merged**